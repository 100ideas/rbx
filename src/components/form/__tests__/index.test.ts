import * as Form from "..";

describe("Form component", () => {
  it("should expose all Form elements", () => {
    expect(Form).toMatchSnapshot();
  });
});
