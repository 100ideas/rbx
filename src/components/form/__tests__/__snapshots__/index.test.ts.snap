// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form component should expose all Form elements 1`] = `
Object {
  "Checkbox": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "checked": false,
      "children": null,
      "disabled": false,
    },
    "render": [Function],
  },
  "Control": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "div",
      "fullwidth": false,
      "iconLeft": false,
      "iconRight": false,
      "loading": false,
    },
    "render": [Function],
  },
  "Field": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Body": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Label": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "div",
      "horizontal": false,
      "multiline": false,
    },
    "render": [Function],
  },
  "Help": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "p",
    },
    "render": [Function],
  },
  "Input": Object {
    "$$typeof": Symbol(react.forward_ref),
    "render": [Function],
  },
  "InputFile": [Function],
  "Label": Object {
    "$$typeof": Symbol(react.forward_ref),
    "render": [Function],
  },
  "Radio": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "checked": false,
      "children": null,
      "disabled": false,
    },
    "render": [Function],
  },
  "Select": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "children": null,
      "disabled": false,
      "loading": false,
      "multiple": false,
      "readOnly": false,
    },
    "render": [Function],
  },
  "Textarea": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "disabled": false,
      "readOnly": false,
      "rows": 4,
    },
    "render": [Function],
  },
}
`;
